src/Command/InfectionCommand.php

```diff
@@ -174,6 +174,12 @@ protected function configure(): void
                 InputOption::VALUE_REQUIRED,
                 'Extra php options for the initial test runner. Will be ignored if --coverage option presented.'
             )
+            ->addOption(
+                'skip-initial-test-suite',
+                null,
+                InputOption::VALUE_NONE,
+                'If --coverage option has been used then this bypasses the initial test suite.'
```

Comment:
`Skips the initial test runs - requires the coverage to be provided via the --coverage option`